menyer/beancrush/index.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Beancrush: Menu</title>
+    <link rel="stylesheet" href="beancrush_style.css">
+</head>
+<body>
+    <header>
+        <div id="header_label">
+            <h1>B E A N C R U S H</h1>
+        </div>
+        <div id="header_image">
+            <img src="img_beans.png" alt="Beancrush logo" width="64">
+        </div>
+    </header>
+    <section id="section_01_lunchoffers">
```

Comment:
When I see a placed id I think: "This must be a special case of styling and perhaps also used with javascript". But looking into your css I cannot see why it is special. Is seems to only be used for conveying semantics. Only use id when everything else does not make sense. Like when you want to easily select it with javascript. 

Try to use:
- Semantic element to convey semantics
- If it is not enough because you need to style it uniquely, use a class (even if there is only one).